codegen-luau/runtime/runtime.lua

```diff
@@ -309,6 +331,10 @@ do
 		end
 	end
 
+	function trunc.num(num)
```

Comment:
I'd also rewrite this as a normal if for the same reason of minimizing Luau syntax.